content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+Jeśli preferujesz **naukę przez praktykę**, sprawdź nasze [praktyczne samouczki](/tutorial/tutorial.html). W tym rozdziale stworzymy w Reakcie od podsatw grę w kółko i krzyżyk. Jeśli nie zajmujesz się tworzeniem gier i kusi Cię, by go pominąć -- najpierw daj mu szansę. Techniki, których nauczysz się w tym poradniku są fundamentami do zbudowania *dowolnej* aplikacji w Reakcie, a zrozumienie ich zapewni Ci dogłębne poznanie tematu.
 
-### Step-by-Step Guide {#step-by-step-guide}
+### Instrukcja krok po kroku {#step-by-step-guide}
 
-If you prefer to **learn concepts step by step,** our [guide to main concepts](/docs/hello-world.html) is the best place to start. Every next chapter in it builds on the knowledge introduced in the previous chapters so you won't miss anything as you go along.
+Jeśli wolisz **naukę konceptów krok po kroku**, zacznij od [przewodnika po głównych pojęciach](/docs/hello-world.html) jest najlepszym miejscem do startu. Każdy kolejny rozdział jest oparty o wiedzę nabytą w poprzednim, więc radzimy niczego nie pomijać.
 
-### Thinking in React {#thinking-in-react}
+### Myślenie po reactowemu {#thinking-in-react}
 
-Many React users credit reading [Thinking in React](/docs/thinking-in-react.html) as the moment React finally "clicked" for them. It's probably the oldest React walkthrough but it's still just as relevant.
+Wielu użytkowników Reacta ceni sobie lekturę rozdziału pt. [Myślenie po reactowemu](/docs/thinking-in-react.html), bo dzięki niemu coś im w końcu "zaskoczyło" w głowie. To prawdopodobnie najstarszy przewodnik po Reakcie, jednakże jest on wciąż aktualny.
 
-### Recommended Courses {#recommended-courses}
+### Rekomendowane kursy {#recommended-courses}
 
-Sometimes people find third-party books and video courses more helpful than the official documentation. We maintain [a list of commonly recommended resources](/community/courses.html), some of which are free.
+Czasem ludzie preferują książki i kursy wideo stworzone przez innych użytkowników, ponieważ dla nich są one bardziej pomocne. Dla takich osób stworzyliśmy [listę rekomendowanych źródeł](/community/courses.html), z których część jest darmowa.
 
-### Advanced Concepts {#advanced-concepts}
+### Zaawansowane pojęcia {#advanced-concepts}
 
-Once you're comfortable with the [main concepts](#main-concepts) and played with React a little bit, you might be interested in more advanced topics. This section will introduce you to the powerful, but less commonly used React features like [context](/docs/context.html) and [refs](/docs/refs-and-the-dom.html).
+Kiedy będziesz się już czuć komfortowo z [podstawowymi pojęciami](#main-concepts), a za sobą będziesz mieć trochę zabawy z kodem w Reakcie, możesz być zainteresowany bardziej zaawansowanymi koncepcjami. Ta sekcja zapozna Cię z przydatnymi, acz rzadziej używanymi funkcjonalnościami jak [konteksty (ang. *contexts*)](/docs/context.html) czy [referencjami (ang. *refs*)](/docs/refs-and-the-dom.html).
```

Comment:
`koncepcjami` - jeżeli przy tym PR: #4 wygra słowo `pojęć` lub `założeń` to tutaj trzeba będzie nanieść tę zmianę aby dokumentacja była spójna.